sources/web/datalab/polymer/components/datalab-toolbar/datalab-toolbar.html

```diff
@@ -0,0 +1,119 @@
+<link rel="import" href="../../components/shared-styles/shared-styles.html">
+
+<link rel="import" href="../../bower_components/app-layout/app-header/app-header.html">
+<link rel="import" href="../../bower_components/app-layout/app-toolbar/app-toolbar.html">
+<link rel="import" href="../../bower_components/iron-icons/iron-icons.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../bower_components/paper-dialog/paper-dialog.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+
+<dom-module id="datalab-toolbar">
+  <template>
+    <style include="datalab-shared-styles">
+      :host {
```

Comment:
I try to keep them roughly in the same order as the elements in the dom, but sometimes I miss something. How important do you think ordering is here?